.github/workflows/ubuntu_clean_meson_build.yml

```diff
@@ -28,10 +28,20 @@ jobs:
       if: env.rebuild == '1'
       run: |
         sudo apt-get update && sudo apt-get install -y libglib2.0-dev libjson-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libunwind-dev googletest gdb
-        sudo apt-get install -y liborc-0.4-dev flex bison libopencv-dev pkg-config python3-dev python3-numpy python3
+        sudo apt-get install -y liborc-0.4-dev flex bison libopencv-dev pkg-config python3-dev python3-numpy python3 libcurl4-openssl-dev
         sudo add-apt-repository -y ppa:nnstreamer/ppa && sudo apt-get update && sudo apt-get install -y ssat libpaho-mqtt-dev
         sudo apt-get install -y valgrind gstreamer1.0-tools gstreamer1.0-plugins-good gstreamer1.0-plugins-base libgtest-dev libpng-dev libc6-dbg binutils-x86-64-linux-gnu-dbg valgrind-dbg
         pip install meson ninja
+    - name: clone and build llama.cpp
+      if: env.rebuild == '1'
+      run: |
+        echo "Current directory: $(pwd)"
+        git clone https://github.com/ggml-org/llama.cpp.git
+        cd llama.cpp
+        cmake -B build -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/lib
```

Comment:
You may install pre-built binaries from https://github.com/ggml-org/llama.cpp/releases

Reply:
Thank you, I've updated.

Reply:
git clone seems better.
Because building nnstreamer requires llama.pc, llama.h, and llamacpp.h, which are not included in the release.

Reply:
The prebuilt zip file does not include the h and pc files. We must manually copy the .so files from the prebuilt binary to /usr/lib/ and the .h files to /usr/include/, then run ldconfig. You can also manually generate the pc files. This method links libllama.so itself, but libllama.so fails to find the other backend libraries (libggml*) it attempts to dynamically load.

error log
```
stderr:
llama_model_load_from_file_impl: no backends are loaded. hint: use ggml_backend_load() or ggml_backend_load_all() to load a backend before calling this function
** (unittest_filter_llamacpp:8385): CRITICAL **: 08:16:27.041: Failed to load model
```